x-pack/plugins/aiops/public/embeddable/index.ts

```diff
@@ -5,5 +5,14 @@
  * 2.0.
  */
 
-export { EmbeddableChangePointChartFactory } from './embeddable_change_point_chart_factory';
-export { type EmbeddableChangePointChartProps } from './embeddable_change_point_chart_component';
+import type { CoreSetup } from '@kbn/core-lifecycle-browser';
+import { registerReactEmbeddableFactory } from '@kbn/embeddable-plugin/public';
+import { EMBEDDABLE_CHANGE_POINT_CHART_TYPE } from '@kbn/aiops-change-point-detection/constants';
+import type { AiopsPluginStart, AiopsPluginStartDeps } from '../types';
+
+export const registerEmbeddable = (core: CoreSetup<AiopsPluginStartDeps, AiopsPluginStart>) => {
```

Comment:
We're going to be adding more embeddables in the near future, so I think this folder should be called `embeddables` and this function `registerEmbeddables`